openstf/pipeline/train_model_sklearn.py

```diff
@@ -135,7 +137,9 @@ def train_model_pipeline(
         train_y,
         eval_set=eval_set,
         early_stopping_rounds=DEFAULT_EARLY_STOPPING_ROUNDS,
+        verbose=0,
```

Comment:
Just wondering what does this do? Would it not be more clear to use `False`? Is the default not `False` anyway?

Reply:
verbosity is [0-3] and specifies how much info is printed during training. Anything other than 0 is too much for practical use;)